### ISSUE TITLE
Can `exclude` be a list? To exclude multiple different files?

### ISSUE THREAD

**Issue by flynneva** [mondeja/mkdocs-include-markdown-plugin#172]
Something like:

```
plugins:
   - include-markdown:
       exclude:
         - LICENSE.md
         - some_other.md
```

**Comment by flynneva** [comment]
Oh...actually I thought the `exclude` feature excluded entire files from being checked by this plugin. Can that be added as a feature here?

**Comment by mondeja** [comment]
Thanks for opening this issue.

Just released [v6.0.0](https://github.com/mondeja/mkdocs-include-markdown-plugin/releases/tag/v6.0.0) which allows to define more flexible patterns for exclusions. I've included the case that you're exposing here as an example (*examples/global-exclude*), check it [here](https://github.com/mondeja/mkdocs-include-markdown-plugin/tree/master/examples/global-exclude).
